scripts/apps/search/components/ItemList.jsx

```diff
@@ -29,6 +31,10 @@ export class ItemList extends React.Component {
         this.setSelectedComponent = this.setSelectedComponent.bind(this);
         this.modifiedUserName = this.modifiedUserName.bind(this);
         this.setNarrowView = this.setNarrowView.bind(this);
+
+        keyboardManager.bind('x', () => {
```

Comment:
I understand, thanks for the explanation. Maybe we could also bind it like the other methods above?
```js
this.multiSelectCurrentItem = this.multiSelectCurrentItem.bind(this);
```
I wonder if that would fix it.... Just to make sure other callers don't run across this weird behaviour.